src/components/NewProjectFlow/flow/SelectAccessions.tsx

```diff
@@ -171,6 +174,34 @@ export default function SelectAccessions(props: SelectAccessionsProps): JSX.Elem
     getSearchFields,
   });
 
+  const entitySpecificFilterConfigs: EntitySpecificFilterConfig[] = useMemo(
+    () => [
+      {
+        label: strings.STATUS,
+        initialSelection: filters.statuses || [],
+        filterKey: 'statuses',
+        options: ACCESSION_2_STATES,
+        renderOption: (value: string | number) => `${value}`,
```

Comment:
use `stateName` from https://github.com/terraware/terraware-web/blob/main/src/types/Accession.ts#L33 for i18n purposes